style.css

```diff
@@ -0,0 +1,163 @@
+body {
+  font-family: 'Poppins', sans-serif;
+  width: 375px;
+  height: 718px;
```

Comment:
- [ ] As suggested by the previous reviewer. It's better to remove all limitations of width and height related to the size of the mobile screen (375px, 327px). That will be helpful for the creation of the desktop version.